src/Stratis.Bitcoin.IntegrationTests/Wallet/SendingTxWithDoubleSpendSpecification.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Xunit;
+
+namespace Stratis.Bitcoin.IntegrationTests.Wallet
+{
+    public partial class SendingTxWithDoubleSpend
+    {
+        [Fact]
+        public void sending_tx_with_double_spend()
```

Comment:
As with the other test, the boundaries need to change here between what is a Given, When and Then, line 3 looks like the When here and line 2 a Given, I'll inline those comments below actually